lib/bolt/bolt_option_parser.rb

```diff
@@ -761,6 +761,14 @@ def initialize(options)
       end
 
       separator "\nDISPLAY OPTIONS"
+      define('--filter FILTER', 'Filter tasks and plans by a matching substring') do |filter|
+        unless /[a-z0-9_:]+/.match(filter)
```

Comment:
Whoops my bad, we actually want to anchor this, `/^[a-z0-9_:]+$/` otherwise it matches as long as there are _any_ of those characters in the string.

Reply:
Gotta love regex. :)